pkg/descheduler/strategies/lownodeutilization.go

```diff
@@ -214,24 +196,26 @@ func getNodeUsage(
 		if len(node.Status.Allocatable) > 0 {
 			nodeCapacity = node.Status.Allocatable
 		}
+		lowResourceThreshold := make(map[v1.ResourceName]*resource.Quantity)
+		for name, percent := range lowThreshold {
+			cap := nodeCapacity[name]
+			lowResourceThreshold[name] = resource.NewMilliQuantity(int64(float64(percent)*float64(cap.MilliValue())*0.01), resource.DecimalSI)
+		}
+		highResourceThreshold := make(map[v1.ResourceName]*resource.Quantity)
+		for name, percent := range highThreshold {
+			cap := nodeCapacity[name]
+			highResourceThreshold[name] = resource.NewMilliQuantity(int64(float64(percent)*float64(cap.MilliValue())*0.01), resource.DecimalSI)
+		}
 
 		nodeUsageList = append(nodeUsageList, NodeUsage{
 			node:    node,
 			usage:   nodeUtilization(node, pods),
 			allPods: pods,
-			// A treshold is in percentages but in <0;100> interval.
+			// A threshold is in percentages but in <0;100> interval.
 			// Performing `threshold * 0.01` will convert <0;100> interval into <0;1>.
 			// Multiplying it with capacity will give fraction of the capacity corresponding to the given high/low resource threshold in Quantity units.
-			lowResourceThreshold: map[v1.ResourceName]*resource.Quantity{
-				v1.ResourceCPU:    resource.NewMilliQuantity(int64(float64(lowThreshold[v1.ResourceCPU])*float64(nodeCapacity.Cpu().MilliValue())*0.01), resource.DecimalSI),
```

Comment:
Cpu is computed in mili compared to memory/pods. Also, [extended resources are expected to be natural numbers](https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/):
```
The API server restricts quantities of extended resources to whole numbers. Examples of valid quantities are 3, 3000m and 3Ki. Examples of invalid quantities are 0.5 and 1500m.
```

Also, as mentioned before, cpu, memory and pods are basic native resources that are taken into account when evicting pods. Thus, all three are required to be set. Also, quantities of all extended resources are natural numbers so I'd expect them to be set optional alongside the basic native resources. Please, keep `lowResourceThreshold` and `highResourceThreshold` as they are and extend the maps only when other resources besides cpu, memory and pods are present.